database.py

```diff
@@ -0,0 +1,26 @@
+import psycopg2
+class DatabaseConnection:
+    def __init__(self):
+        try:
+            self.connection = psycopg2.connect(
```

Comment:
change this line to 
```python
self.connection = psycopg2.connect("dbname=maintenence_tracker user=admin password=admin host=localhost port=5432 ")
```
Or
```python
self.connection = psycopg2.connect(dbname='maintenence_tracker',user='admin',password='admin',host='localhost',port='5432')
```

Reply:
since you will be using the configs in multiple places, You can consider creating them in a dictionary some where and importing them i,e
> In a file say configs.py
```python
db_configs = {
    'dbname': 'maintenence_tracker',
    'user': 'postgres',
    'host': 'localhost',
    'port': 5432
}
```
Use them when creating a connection 
```python
from configs import db_configs

pyscopg2.connect(**db_configs)
```